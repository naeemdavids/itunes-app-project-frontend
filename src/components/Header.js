import React from 'react'

//Simple component for the header section of the app.
function Header() {
  return (
    <h1 className='text-info text-center bg-dark'>iTunes Music App</h1>
  )
}

export default Header;