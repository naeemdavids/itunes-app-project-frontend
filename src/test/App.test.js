import React from "react";
//import App from "../App";
import renderer from 'react-test-renderer';
//import Song from "../components/Song";
import Header from "../components/Header";

it("App Snapshot test", () => {
    const tree = renderer
    .create(<Header/>)
    expect(tree.toJSON).toMatchSnapshot();
})