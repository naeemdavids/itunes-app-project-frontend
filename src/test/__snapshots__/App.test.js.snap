// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot test 1`] = `[Function]`;
